1.3/spec.md

```diff
@@ -0,0 +1,359 @@
+**the [global-mercator](http://wiki.osgeo.org/wiki/Tile_Map_Service_Specification#global-mercator) (aka Spherical Mercator) profile is assumed**.
+
+Note that in the TMS tiling scheme, the Y axis is reversed from the coordinate system commonly used in the URLs
+to request individual tiles, so the tile commonly referred to as 11/327/791 is inserted as
+`zoom_level` 11, `tile_column` 327, and `tile_row` 1256, since 1256 is 2^11 - 1 - 791.
+
+The `tile_data blob` column MUST contain the raw binary image or vector tile data
+for the associated tile.
+
+### Grids
+
+_See the [UTFGrid specification](https://github.com/mapbox/utfgrid-spec) for
+implementation details of grids and interaction metadata itself: the MBTiles
+specification is only concerned with storage._
+
+#### Schema
+
+The database MAY have tables named `grids` and `grid_data`.
+
+The `grids` table MUST contain four columns, named `zoom_level`, `tile_column`,
+`tile_row`, and `grid`. A typical create statement for the `grids` table:
+
+    CREATE TABLE grids (zoom_level integer, tile_column integer, tile_row integer, grid blob);
+
+The `grid_data` table MUST contain five columns named `zoom_level`, `tile_column`,
+`tile_row`, `key_name`, and `key_json`. A typical create statement for the `grid_data` table:
+
+    CREATE TABLE grid_data (zoom_level integer, tile_column integer, tile_row integer, key_name text, key_json text);
+
+#### Content
+
+The `grids` table contains UTFGrid data, compressed in `gzip` format.
+
+The `grid_data` table contains grid key to value mappings, with values encoded
+as JSON objects.
+
+## Vector tileset metadata
+
+As mentioned above, Mapbox Vector Tile tilesets MUST include a `json` row in the `metadata` table
+to summarize what layers are available in the tiles and what attributes are available for the
+features in those layers.
+
+### Vector_layers
+
+The `json` row MUST contain the string representation of a JSON object.
+
+The object MUST contain a `vector_layers` key, whose value is an array of layers.
+Each layer is a JSON object that MUST contain the following key-value pairs:
+
+* `id`: The layer ID, which is referred to as the `name` of the layer in the [Mapbox Vector Tile spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#41-layers).
+* `fields`: A JSON object whose keys and values are the names and types of attributes available in this layer.
+The type MUST be the string `"Number"`, `"Boolean"`, or `"String"`.
+Attributes whose type varies between features SHOULD be listed as `"String"`.
+
+Each layer object MAY also contain the following key-value pair:
+
+* `description`: A human-readable description of the layer's contents.
+
+Each layer object MAY also contain the following key-value pair:
+
+* `minzoom`: The lowest zoom level whose tiles this layer appears in.
+* `maxzoom`: The highest zoom level whose tiles this layer appears in.
+
+The `minzoom` SHOULD be greater than or equal to the tileset's `minzoom`,
```

Comment:
Why SHOULD here instead of MUST?

Reply:
You're right, there's no good reason that a layer's zoom levels should go beyond the tileset's.

Reply:
You might also add wording such as "If `minzoom` exists..."